doc.go

```diff
@@ -167,12 +168,12 @@ you are looking to connect. Since the driver name is not needed, you can optiona
 on startup. To do this, set `GOSNOWFLAKE_SKIP_REGISTERATION` in your environment. This is useful you wish to
 register multiple verions of the driver.
 
-Note: GOSNOWFLAKE_SKIP_REGISTERATION should not be used if sql.Open() is used as the method
+Note: `GOSNOWFLAKE_SKIP_REGISTERATION` should not be used if sql.Open() is used as the method
```

Comment:
Side note: `REGISTERATION` is spelled wrong 😉 It seems to be that way in the code as well, so didn't want to change it.